src/pybind/mgr/dashboard/module.py

```diff
@@ -137,6 +137,7 @@ def _configure(self):
                 'text/html', 'text/plain',
                 # We also want JSON and JavaScript to be compressed
                 'application/json',
+                'application/vnd.ceph.api.v1.0+json',
```

Comment:
Mmm... I wasn't aware of this... Good news is that gzip tool supports [generic MIME subtypes](https://github.com/cherrypy/cherrypy/blob/2fc78dc9af3e556829ce69d3e1311e5629200882/cherrypy/lib/encoding.py#L336):
```suggestion
                'application/*+json',
```